lib/wasix/src/runtime/resolver/wapm_source.rs

```diff
@@ -144,6 +208,149 @@ fn decode_summary(pkg_version: WapmWebQueryGetPackageVersion) -> Result<PackageS
     })
 }
 
+/// A local cache for package queries.
+#[derive(Debug, Clone)]
+struct FileSystemCache {
+    cache_dir: PathBuf,
+    timeout: Duration,
+}
+
+impl FileSystemCache {
+    const DEFAULT_TIMEOUT: Duration = Duration::from_secs(10 * 60);
```

Comment:
So, cache is valid 10 min.
Could cache be disabled from command line also?

Reply:
tere are indeed many cache, but not all are "internet" cache. For example, I don't think wapm cache should be mixed with compiler cache (there is a `--disable-cache` for that already).
But some control over the caches layer on command line would be nice for a later version.